Examples/snippets.lua

```diff
@@ -279,12 +280,14 @@ ls.add_snippets("all", {
 		})
 	),
 	-- The delimiters can be changed from the default `{}` to something else.
-	s("fmt4", fmt("foo() { return []; }", i(1, "x"), { delimiters = "[]" })),
+	s("fmt4", fmt("foo() { return (); }", i(1, "x"), { delimiters = "()" })),
```

Comment:
I don't really like this change, in almost all languages, it would be very annoying to use `()`, `[]` just make more sense since they are more rarely used (as you have realised :D)

What do you think of, instead, amending the comment above, stating that `[]` is also available, directly, via `fmts`?

Reply:
Certainly this is not good because it is not common.
I changed it because I didn't want to duplicate fmts, but I'll change it back.